benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadWorker.java

```diff
@@ -104,7 +104,8 @@ public static void main(String[] args) throws Exception {
               + "\n  --driver_port=<port>"
               + "\n    Port to expose grpc.testing.WorkerService, used by driver to initiate work."
               + "\n  --server_port=<port>"
-              + "\n    Port to start load servers on. Defaults to any available port");
+              + "\n    Port to start load servers on, if not specified by the server config message."
```

Comment:
Off by one. It is 101 characters.

Reply:
It is also 81 characters to the terminal

Reply:
Aargh, sorry for not checking that carefully.
Should be fixed now.